app/Models/Post.php

```diff
@@ -218,22 +218,35 @@ public function toBlinkPayload()
         // Blink expects quantity to be a number.
         $quantity = $this->quantity === null ? 0 : $this->quantity;
 
+        // Array values aren't supported in Customer.io event attributes.
+        $campaign_cause = implode(',', (array) $this->signup->campaign['cause']);
```

Comment:
An alternative method would be to just obtain the original DB value without the [accessor formatting](https://github.com/DoSomething/rogue/blob/c79e6c145f6f3a923246b64c78bf6772cebd6d8e/app/Models/Campaign.php#L84-L94). (I thought it was a drop clunkier, but the advantage I guess is that we don't need to do explicit formatting on the value and just grab it as-is from the campaign?)
```suggestion
        $campaign_cause = (string) optional($this->signup->campaign)->getAttributes()['cause'];
```

Reply:
That `optional()` helper is amazing who knew???? 

The 2nd option (straight from DB) seems like there is slightly less room for error but I agree that the way you have it now is cleaner, so your call!

Reply:
Hmm, ok, in that case, I kinda feel like going with the 2nd option so it's less error-prone rather than cleaner!